src/main/groovy/org/xitikit/iqserver/gradle/IqServerPlugin.groovy

```diff
@@ -11,18 +11,17 @@ import static org.xitikit.iqserver.gradle.IqServerExtensionProperties.*
 class IqServerPlugin implements Plugin<ProjectInternal> {
     @Override
     void apply(@Nonnull final ProjectInternal project) {
-
         buildIqScan(
             project,
             createIqServerData(project))
     }
 
     private static IqServerData createIqServerData(@Nonnull final ProjectInternal project) {
-        return project.getExtensions().
-            create(
-                IQ_SERVER,
-                IqServerExtension
-            ).iqServerData
+        project.extensions.create(
+            IQ_SERVER,
+            IqServerExtension,
+            new IqServerData()
```

Comment:
This is nice. I didn't realize I could pass constructor parameters here. Would it also be possible to sue named parameters?

Reply:
Yeah if I had instantiated new IqServerData like:

`IqServerData data = new IqServerData()`

I could have just passed in `data`. In this case I inlined it since I didn't immediately see a need to do anything else.